include/swift/AST/Module.h

```diff
@@ -744,7 +748,7 @@ class FileUnit : public DeclContext {
   // Only allow allocation of FileUnits using the allocator in ASTContext
   // or by doing a placement new.
   void *operator new(size_t Bytes, ASTContext &C,
-                     unsigned Alignment = alignof(FileUnit));
+                     unsigned Alignment = alignof(FileUnit&));
```

Comment:
Specific citation allowing this (which says nothing to whether MSVC implements it): "Within the class _member-specification_, the class is regarded as complete within function bodies, **default arguments,** and _brace-or-equal-initializers_ for non-static data members (including such things in nested classes)." (C++11 [class.mem]p2)